deploy-manage/security/azure-private-link-traffic-filters.md

```diff
@@ -114,85 +120,78 @@ Follow these high-level steps to add Private Link rules to your deployments.
 3. Copy the value of the **properties.resourceGUID** property.
 
 :::{image} /deploy-manage/images/cloud-ec-private-link-azure-json-view.png
-:alt: Private endpoint JSON View
+:alt: Private endpoint JSON view
 :screenshot:
 :::
 
 :::{image} /deploy-manage/images/cloud-ec-private-link-azure-properties.png
-:alt: Private endpoint Properties
+:alt: Private endpoint properties
 :screenshot:
 :::
+% fix me
 
 
-### Create rules using the Private Link Endpoint Resource Name and Resource ID [ec-azure-create-traffic-filter-private-link-rule-set]
+### Create a policy using the Private Link Endpoint resource ID [ec-azure-create-traffic-filter-private-link-rule-set]
 
-When you have your private endpoint name and ID, you can create a Private Link traffic filter rule set.
+When you have your private endpoint ID, you can create a private connection policy.
 
 ::::{note}
-The Private Link connection will be approved automatically after the traffic filter is created.
+The Private Link connection will be approved automatically after the private connection policy is created.
 ::::
 
 
-1. From the **Account** menu, select **Traffic filters**.
-2. Select **Create filter**.
-3. Select **Private link endpoint**.
-4. Create your rule set, providing a meaningful name and description.
-5. Select the region for the rule set.
-6. Enter your Private Endpoint Resource Name and Resource ID.
-7. Select if this rule set should be automatically attached to new deployments.
-
-    ::::{note}
-    Each rule set is bound to a particular region and can be only assigned to deployments in the same region.
-    ::::
-
-8. (Optional) You can [claim your Private Endpoint Resource Name and Resource ID](/deploy-manage/security/claim-traffic-filter-link-id-ownership-through-api.md), so that no other organization is able to use it in a traffic filter ruleset.
+:::{include} _snippets/network-security-page.md
+:::
+4. Select **Private connection**.
+5. Select the resource type that the private connection will be applied to. Currently, only hosted deployments are supported.
+6.  Select the cloud provider and region for the private connection. 
+   
+    :::{tip}
+    Network security policies are bound to a single region, and can be assigned only to deployments in the same region. If you want to associate a policy with resources in multiple regions, then you have to create the same policy in all the regions you want to apply it to.
+    :::
+7.  Under **Connectivity**, select **Privatelink**.
+8.  Under **VPCE filter**, enter your Private Endpoint resource ID.
```

Comment:
azure requires both the resource id + name. need to clarify what the UI looks like here